### PR TITLE
Baba la gumbala with instructions

### PR DESCRIPTION
**Pull request by shiundumichael**

### CODE REVIEW

**Comment by barbra2020** [comment]
added a full colon on One:Play.....
changed the timing from 0:42 to 0:43, also added Two:You.....and 0:43 to 0:44
time from 41 to 43
@shiundumichael I have made changes to your file, please merge the PR I will open to your repo.

File: src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts [hunk 1]
```diff
@@ -200,7 +200,7 @@ export const Baba_La_Gumbala_Instructions: Song = {
             "endTimeSecond": "36.59"
         },
         {
-            "lyric": "You sing they sing",
+            "lyric": "Two,You sing they sing",
```
Comment (barbra2020) [inline src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts:203]:
```suggestion
            "lyric": "Two: You sing they sing",
```

Comment (barbra2020) [reply src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts:203]: On line 207 add - 

`"startTimeSecond": "40",`

On line 209 add- 

`"endTimeSecond": "43"`

And on line

File: src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts [hunk 2]
```diff
@@ -425,7 +425,7 @@ export const Baba_La_Gumbala_Instructions: Song = {
             "endTimeSecond": "40"
         },
         {
-            "lyric": "Add fun actions like making a big pot of stone soup,chopping incredient,stirring pot and testing the soup",
+            "lyric": "Three,Add fun actions like making a big pot of stone soup,chopping the ingredient,stirring pot and tasting the soup",
```
Comment (barbra2020) [inline src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts:428]:
```suggestion
            "lyric": "Three: Add fun actions like making a big pot of stone soup,chopping the ingredient,stirring pot and tasting the soup",
```

Comment (barbra2020) [reply src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts:428]: On line 432 add- 

`"startTimeSecond": "43",`

File: src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts [hunk 3]
```diff
@@ -434,7 +434,7 @@ export const Baba_La_Gumbala_Instructions: Song = {
             "endTimeSecond": "56"
         },
         {
-            "lyric": "Sing all together",
+            "lyric": "Four,Sing all together",
```
Comment (barbra2020) [inline src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts:437]:
```suggestion
            "lyric": "Four: Sing all together",
```

**Comment by maxwellfundi** [comment]
@michaelclapham  this should now be looking good with the new changes done :-)

File: src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts [hunk 4]
```diff
@@ -587,4 +587,4 @@ export const Baba_La_Gumbala_Instructions: Song = {
             "endTimeSecond": "36"
         }
     ]
-};
\ No newline at end of file
+}
```
Comment (maxwellfundi) [inline src/app/lyrics/songs/baba-la-gumbala-with-instructions.ts:590]:
```suggestion
};
```

**Pull request merged**